_posts/2018-03-01-core-team-announcement.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Core Team Announcement
+summary: Small elaboration on the Core Team's role and an announcement on its membership
+tags:  [monero, core]
+author: The Core Team
+---
+
+The year 2018 is well underway, with many exciting new technologies on the horizon for Monero. Technology is not the only thing that needs to adapt if Monero is to stay at the forefront of privacy and blockchain technologies, however. With that in mind, the Core Team would like to remind the community about their role in regards to Monero followed by an announcement regarding the membership of the Core Team. 
+
+The role of the Core Team can often be a confusing topic, especially with the various leadership structures employed by other cryptocurrencies. As the technology for a completely trustless society is not yet a reality, all projects, even decentralized ones, benefit from having individuals that can carry out tasks that require trust. A quick example of this is owning a domain name (like getmonero.org) or maintaining servers (on which getmonero.org is hosted). Ultimately, there is not yet a way for a domain name to be owned by a decentralized community, so someone has to be the name holding this domain. It's in scenarios like this where the Core Team are able to work on bridging the decentralized world we are trying to bring about with the centralized reality in which we are currently living.
+

```

Comment:
>Ultimately, there is not yet a way for a domain name to be owned by a decentralized community

Centralization is a requisite for ownership. One can never "own" something while being decentralized because ownership requires at least one instance of authority. Authority requires dominance which requires followers which, in turn, forms a centralized body. Ergo, multiple authorities equate to multiple points of centralization.

>domain name to be owned by a decentralized community

There is a side-step for the domain issue and it's called kovri/i2p multi-homing:

>>Destinations may be hosted on multiple routers simultaneously, by using the same private and public keys. As both instances will periodically publish their signed LeaseSets to the floodfill peers, the most recently published LeaseSet will be returned to a peer requesting a database lookup. As LeaseSets have (at most) a 10 minute lifetime, should a particular instance go down, the outage will be 10 minutes at most, and generally much less than that. The multihoming function has been verified and is in use by several services on the network.

In other words, you and Nth other people can all "own" the same address after you distribute your site's private keys to said people (not advised unless you trust them or don't mind the potential that they could host something you don't like).